test/benchmarks/run.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+time solc --via-ir --optimize --bin chains.sol
```

Comment:
This doesn't use the per step benchmarking you implemented, but we could at least add another run without `--via-ir` and then compare and output the runtimes to see where we're at roughly.

Reply:
It does when you compile it correctly :)

Comparing it with non-via-ir would destroy morale, so we rather not do it :D

Reply:
Beatings will continue until morale improves :D Anyway, it would nonetheless be nice if we were to at least run this in a loop several times and then take the average runtime (mean) whilst also outputting the lowest and highest (or standard deviation) so at least we're not taking the one and only run as the representative data point.